### ISSUE TITLE
Question: Key Shared sample.

### ISSUE THREAD

**Issue by JimmyBjorklund** [apache/pulsar-client-node#350]
Looking for a sample on how to setup a producer and consumers using key shared. Using the pulsar node client, the only samples i can find are using java and thy have a key attribute that is not in the node api what i can see.

In the ProducerMessage there is a orderingKey, is this the corresponding key that will be used? When setting the ConsumerConfig.subscriptionType = KeyShared

Thanks for any input.
/J

**Comment by shibd** [comment]
> In the ProducerMessage there is a orderingKey, is this the corresponding key that will be used? When setting the ConsumerConfig.subscriptionType = KeyShared

Yes. You can refer to: https://pulsar.apache.org/docs/3.1.x/concepts-messaging/#key_shared

It works with `orderingKey` or `partitionKey` .
